notify/Cargo.toml

```diff
@@ -14,6 +14,6 @@ ckb-app-config  = { path = "../util/app-config", version = "= 0.120.0-pre" }
 ckb-types = { path = "../util/types", version = "= 0.120.0-pre" }
 ckb-stop-handler = { path = "../util/stop-handler", version = "= 0.120.0-pre" }
 ckb-async-runtime = { path = "../util/runtime", version = "= 0.120.0-pre" }
-tokio = { version = "1", features = ["sync"] }
+tokio = { version = "1", features = ["sync", "process"] }
```

Comment:
this feature should add `cfg(not(target_family = "wasm"))`

Reply:
Changed. Please review whether the Cargo.toml is correct.

Reply:
Correct, it can also be written like this, without having to specify what is needed under wasm:
```toml
[dependencies]
tokio = { version = "1", features = ["sync"] }

[target.'cfg(not(target_family = "wasm"))'.dependencies]
tokio = { version = "1", features = ["process"] }
```